next/components/molecules/SearchPageNew/AdvancedSearchNew.tsx

```diff
@@ -0,0 +1,75 @@
+import { Typography } from '@bratislava/component-library'
+import Button from '@components/forms/simple-components/Button'
+import { useTranslations } from 'next-intl'
+import { Dispatch, SetStateAction } from 'react'
+import { Input, SearchField, Text } from 'react-aria-components'
+import { twMerge } from 'tailwind-merge'
+
+import SearchIcon from '../../../assets/images/search-icon.svg'
```

Comment:
Alway use absolute imports. For all assets (images, icons...) we write barrel file (index.ts), so al icons are imported always with the same name. So the import should look like
```
import { SearchIcon } from '@assets/ui-icons'
```